crates/bevy_render/src/color/mod.rs

```diff
@@ -47,43 +47,81 @@ pub enum Color {
 }
 
 impl Color {
+    /// <div style="background-color:rgb(240, 248, 255); width: 10px; padding: 10px; border: 1px solid;"></div>
     pub const ALICE_BLUE: Color = Color::rgb(0.94, 0.97, 1.0);
```

Comment:
```suggestion
    /// <div style="background-color:rgb(94%, 97%, 100%); width: 10px; padding: 10px; border: 1px solid;"></div>
    pub const ALICE_BLUE: Color = Color::rgb(0.94, 0.97, 1.0);
```
Can you put the colors with percentages? It's easier to have the same values that way

Reply:
ok